src/schemas/InvoiceObject.yaml

```diff
@@ -36,6 +36,15 @@ properties:
     format: 'date'
     description: The date when the invoice was issued. It is provided in the ISO 8601 date format.
     example: '2022-04-30'
+  payment_due_date:
+    type: string
+    format: 'date'
+    description: The payment due date for the invoice, specified in the ISO 8601 date format.
+    example: '2022-04-30'
+  net_payment_term:
```

Comment:
Is `net_payment_term` on the invoice? 🤔 

I though it was on customer and organization...

Reply:
Yes it is. I missed that and added it back in this PR https://github.com/getlago/lago-api/pull/1205/files#diff-007abcbae923cc2024a722ecf10691df416ccf540285ed2d84aa33e05ac4d9c9